docs/building-spark.md

```diff
@@ -27,7 +27,7 @@ license: |
 ## Apache Maven
 
 The Maven-based build is the build of reference for Apache Spark.
-Building Spark using Maven requires Maven 3.8.7 and Java 8.
+Building Spark using Maven requires Maven 3.8.8 and Java 8.
```

Comment:
Do we think 3.8.8 is required? I'd imagine 3.8.7 still works fine

Reply:
Based on the comments in https://github.com/apache/spark/pull/22781, I think we should change it to

```
Building Spark using Maven requires Maven 3.8.8 and Java 8/11/17.
```

because only the above versions were tested

Reply:
I think Maven 3.8.7 should work well, but w/o CI verification, it may break silently anytime.